### PR TITLE
Dev joris#439

### PR DESCRIPTION
**Pull request by JorisBassle**
Extra option for admin. Deleting the group deletes the chat for everyone.
image

The admin has to select the next admin before leaving the group. Before it was just the 1st contact that became the next admin.
image

Also PR in the backend: dev-joris#439

### CODE REVIEW

File: src/store/chatStore.ts [hunk 1]
```diff
@@ -569,6 +569,7 @@ const updateContactsInGroup = async (groupId: string, contact: Contact, type: Sy
             message: msg,
             adminLocation,
             contact,
+            nextAdmin: nextAdmin
```
Comment (EdwardKerckhof) [inline src/store/chatStore.ts:572]: Can just be `nextAdmin`

**Review by EdwardKerckhof — APPROVED** [review]
Looks good just one small thing on line 572 in chatStore.ts. (See comment)

**Comment by JensMeire** [comment]
Where is the code of the pop up or does this exist?

**Review by JensMeire — APPROVED** [review]
First resolve backend code, then u can merge this

**Pull request merged**